README.md

```diff
@@ -65,11 +65,40 @@ the API. The test server does not have any authentication mechanism. Any
 username you submit on the login form will be authenticated and can post a
 proof.
 
+
+## Example config
+
+Here is an example configuration if you were to use this library. You should
+check out [the
+documentation](https://keybase.io/docs/proof_integration_guide#1-config) for
+the complete description of what's going on here.
+
+```json
+{
+  "version": 1,
```

Comment:
i thought version was for internal use?

Reply:
nope it's required so configs can't be replayed when signing